photon-lib/src/main/java/org/photonvision/timesync/TimeSyncSingleton.java

```diff
@@ -35,12 +35,11 @@ public class TimeSyncSingleton {
     public static boolean load() {
         if (INSTANCE == null) {
             try {
-                if (!PhotonTargetingJniLoader.load()) {
-                    return false;
-                }
-            } catch (UnsatisfiedLinkError | IOException e) {
+                RuntimeLoader.loadLibrary("photontargetingJNI");
+            } catch (IOException e) {
+                // Don't want to return early. We want to create the TimeSyncServer so the program crashes
+                // because we need it in order to function.
```

Comment:
Hopefully people don't try catch this. Unsure how defensive we need to be but for now ship it